src/provider/model/FinEnvironment.ts

```diff
@@ -61,6 +61,15 @@ export class FinEnvironment extends AsyncInit implements Environment {
     private _windowsCreated: number = 0;
     private readonly _seenWindows: {[id: string]: SeenWindow} = {};
 
+    public async isRunning(appInfo: Application): Promise<boolean> {
+        // TODO: This can be simplified once we change how we map between OpenFin apps and directory entries [SERVICE-617]
```

Comment:
617 is currently in review. It's probably worth liaising with Noyan to make sure this gets resolved either here or there so we don't end up with a dangling `TODO`

Reply:
TODO resolved now that 617 is in

✅